libs/ballot-interpreter-nh-next/src/timing_marks.rs

```diff
@@ -113,10 +115,47 @@ pub struct TimingMarkGrid {
     /// Areas of the ballot card that contain shapes that may be timing marks.
     pub candidate_timing_marks: Vec<Rect>,
 
+    /// Mapping of grid positions to their pixel coordinates in the normalized image.
+    pub pixel_points: PixelPoints,
+
     /// Metadata from the ballot card bottom timing marks.
     pub metadata: BallotPageMetadata,
 }
 
+fn compute_pixel_points(geometry: &Geometry, complete_timing_marks: &Complete) -> PixelPoints {
+    let mut pixel_points: PixelPoints =
+        vec![
+            vec![Point::new(0.0, 0.0); geometry.grid_size.width as usize];
+            geometry.grid_size.height as usize
+        ];
```

Comment:
Goal here was to pre-allocate the vectors since we know the needed size. Not actually sure if it's worth doing, or if this is an effective way to do it.

Reply:
Sweet! I gotta get used to these zero-cost abstractions 😄